azure/storage/queueservice.py

```diff
@@ -82,6 +83,16 @@ def __init__(self, account_name=None, account_key=None, protocol='https',
         sas_token:
             Optional. Token to use to authenticate with shared access signature.
         '''
+        if connection_string is not None:
+            connection_params = StorageConnectionParameters(connection_string)
+            account_name = connection_params.account_name
+            account_key = connection_params.account_key
+            protocol = connection_params.protocol
+            host_base = connection_params.host_base_blob
+            dev_host = connection_params.dev_host_blob
```

Comment:
Thanks for your careful review and sorry about the foolish copy-paste bug. Fixed in the next commit.